src/Order/CustomerOrder.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Vendor\Service\Order;
+
+class CustomerOrder extends OrderInf
+{
+    public function description()
+    {
+        echo "Mr/Ms {$this->customer->getName()}.You recently have made an order.<br>
```

Comment:
don't echo in classes. Please return this string and echo in your main.php. You can call this method getDescription